packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -42,11 +43,17 @@ class SingleChildScrollView extends StatelessWidget {
     this.scrollDirection: Axis.vertical,
     this.reverse: false,
     this.padding,
+    this.primary: false,
```

Comment:
Yes, the scrolling classes are missing many dartdocs.  It's on my list for this week.